README.md

```diff
@@ -256,6 +256,10 @@ Or a property can reference a subschema residing in a schema document's [$defs](
 
 In order to promote stylistic consistency and allow for reuse of concepts, `object` and `enum` types should be defined in either as a top level schema document or as a subschema in a schema document's `$defs`.
 
+Each schema has an optional [`$id`](https://json-schema.org/understanding-json-schema/basics#declaring-a-unique-identifier) defining a unique identifier for the schema. Schemas and subschemas can be referenced in other schemas using the [`$ref`](https://json-schema.org/understanding-json-schema/structuring#dollarref) keyword, which resolves against the schema's [base URI](https://json-schema.org/understanding-json-schema/structuring#base-uri) (i.e. the `$id` of the top level schema document) using [JSON pointer](https://json-schema.org/understanding-json-schema/structuring#json-pointer) syntax.
+
+All top level schema documents should contain a `$id` field following the form `https://opentelemetry.io/otelconfig/<name>.json`. Subschemas nested in `$defs` should not include an `$id`, and should be resolved using `$ref`s relative to their top level schema document.
```

Comment:
should we have a version identifier in this path?

Reply:
I think we can avoid this by treating `https://opentelemetry.io/otelconfig/opentelemetry_configuration.json` as the latest, and separately maintaining specific versions at different paths. E.g.:

- https://opentelemetry.io/otelconfig/opentelemetry_configuration.json
- https://opentelemetry.io/otelconfig/v1_0_rc3/opentelemetry_configuration.json
- https://opentelemetry.io/otelconfig/v1_0/opentelemetry_configuration.json
- https://opentelemetry.io/otelconfig/v1_1/opentelemetry_configuration.json

The tooling which maintains the non-latest version could replace `$id` with the the correct path. 

If you feel like this is going to burn us, please do open an issue so we can make sure to resolve before #161.